docker-compose/generate-config.py

```diff
@@ -41,10 +41,10 @@
       - MARIADB_AUTO_UPGRADE=1
     network_mode: host
     healthcheck:
-      test: ['CMD', "mariadb-admin", "--password=password", "--protocol", "tcp", "ping"]
+      test: ['CMD', "healthcheck.sh", "--connect", "--innodb_initialized"]
     volumes:
       - ./mariadb:/var/lib/mysql:rw
```

Comment:
This should probably go to `/srv/mariadb` or somewhere else but should be outside docker-compose IMO.

Reply:
Changed in https://github.com/MariaDB/buildbot/pull/655/commits/05c7c5fc93db1182f610d33336bb27b4cc9c0c8f

I don't think we need to update rsync.exclude, we're deploying in /srv/{dev,prod}.

What's the approach to make this switch on DEV?
My plan is:
 -> stop mariadb container
 -> copy the datadir to /srv/mariadb/
 -> merge this PR

Sounds OK?
@fauust